documentation/docs/practical-sessions/initializing-wis2box.md

```diff
@@ -6,7 +6,7 @@ title: Initializing wis2box
 
 !!! abstract "Learning outcomes"
 
-    By the end of this session you will be able to:
+    By the end of this practical session, you will be able to:
 
     * Run the `wis2box-create-config.py` script to create the initial configuration
```

Comment:
* or - for the list? I'm not sure it makes a difference but we use - elsewhere.

This is also an example of where we have capitalised the list items